Formula/multi-git-status.rb

```diff
@@ -0,0 +1,18 @@
+class MultiGitStatus < Formula
+  desc "Show uncommitted, untracked and unpushed changes for multiple Git repos"
+  homepage "https://github.com/fboender/multi-git-status"
+  url "https://github.com/fboender/multi-git-status/archive/refs/tags/2.0.tar.gz"
+  sha256 "13ce21fc087354cd7e0fd16f332bcff7e8c42c0315d3f27803159926aff3360f"
+  license "MIT"
+
+  def install
+    # This is what the included 'install.sh' script does, except that
+    # we use Homebrew's preferred location for 'man1'.
+    bin.install "mgitstatus"
+    man1.install "mgitstatus.1"
+  end
+
+  test do
+    system "#{bin}/mgitstatus", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Admittedly this is less important for shell scripts, but I'd still like to see fewer `--version` tests in formulae, because other contributors use these as examples for writing tests for new formulae.

Reply:
Hmm, testing anything real would necessitate being in a Git repo of known state. Would it be acceptable to do `git init` in the test directory and check running of the shell script?

Reply:
Yes, that's fine.

Reply:
I'm having a little trouble with it - I tried using the following:
```
  test do                                                                                                                              
    Dir.mkdir("test-repo")                                                                                                             
    Dir.chdir("test-repo"){                                                                                                            
      system "git", "init"                                                                                                             
    }                                                                                                                                  
    assert system("#{bin}/mgitstatus 2>&1 | grep \"Uncommitted changes\"")                                                             
  end                                                                                                                                  
```
but I'm ending up with the error `Minitest::Assertion: Expected nil to be truthy.`  I'm not exactly sure what I'm doing with these assertions, anything obvious jump out?  (`grep` exit status is 0 for a match, 1 for non-match.)

Reply:
This should accomplish what you're trying to do.

```
test do
  mkdir "test-repo" do
    system "git", "init"
    assert_match "Uncommitted changes", shell_output("#{bin}/mgitstatus 2>&1")
  end
end
```

Reply:
Thanks, I've pushed a change with this idea.  It actually goes outside the directory, to exercise the "m" part of mgitstatus.